src/test/scala/com/eclipsesource/schema/SchemaValidatorSpec.scala

```diff
@@ -165,39 +167,37 @@ class SchemaValidatorSpec extends PlaySpecification {
       }
     }
 
-  "should resolve references on the classpath via UrlResolver" in {
-    val validator = SchemaValidator().addUrlResolver(ClasspathUrlResolver())
+  "should resolve references on the classpath with ClasspathUrlResolver" in {
+    val validator = SchemaValidator().addUrlResolver(ClasspathUrlProtocolHandler())
```

Comment:
I'd recommend calling this method `addUrlProtocolHandler(handler: UrlProtocolHandler)` instead of `addUrlResolver(resolver: UrlProtocolHandler)` for consistency. The test description may need to be updated also to remove "resolver".

On that note, you might consider opening up the input type to be a `URLStreamHandler`, like its relative counterpart. That way users could override the default protocol-less handler, or add multiple to try in sequence. This may take more work though.